src/index.js

```diff
@@ -1,23 +1,38 @@
-// import _ from 'lodash';
 import './style.css';
+// eslint-disable-next-line import/no-cycle
+import TodoList, { inputTodo } from './module/updateStatus.js';
+import { editDescription, clearComoleted, render } from './module/addRemove.js';
 
-const todoListContianer = document.querySelector('.list');
+const TodoListObj = new TodoList();
 
-const todoList = [
-  { description: 'Wake up on time', completed: false, index: 0 },
-  { description: 'Complete all the quests of the curriculum', completed: false, index: 1 },
-  { description: 'Go to bed early to rest', completed: false, index: 2 },
+export default function Starter() {
+  const threeDots = document.querySelectorAll('li');
+  threeDots.forEach((dotValue, index) => {
+    const dot = dotValue.querySelector('.dots');
+    if (dot) {
+      const newDot = dot.cloneNode(true);
+      dotValue.replaceChild(newDot, dot);
+      newDot.addEventListener('click', () => {
+        editDescription(dotValue, index, TodoListObj);
+        Starter();
+      });
+    }
+  });
+}
 
-];
+inputTodo.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    TodoListObj.addTodo();
+    render(TodoListObj.list, TodoListObj);
+    Starter();
+  }
+});
```

Comment:
- [ ] **Required:** Excellent job implementing the add function using the `Enter` key. In the same vein, also implement the functionality using the Icon beside the input box.
![image](https://user-images.githubusercontent.com/27709832/150237061-c6019068-a180-4e63-bd85-b9df9efd624d.png)